privaterelay/settings.py

```diff
@@ -607,7 +607,11 @@ def set_index_cache_control_headers(
 SOCIALACCOUNT_PROVIDERS = {
     "fxa": {
         # Note: to request "profile" scope, must be a trusted Mozilla client
-        "SCOPE": ["profile", "https://identity.mozilla.com/account/subscriptions"],
+        "SCOPE": [
+            "profile",
+            "https://identity.mozilla.com/account/subscriptions",
+            "https://identity.mozilla.com/apps/relay",
```

Comment:
Note to self: In my dev environment, I was able to create a new user with this scope against the stage server. When I logged in with an existing user, the user got the `../apps/relay` scope as well.